src/Microsoft.DotNet.Build.Tasks.Packaging/src/FrameworkUtilities.cs

```diff
@@ -36,7 +36,8 @@ public static bool IsGenerationMoniker(NuGetFramework dependencyGroup)
         }
         public static bool IsPortableMoniker(NuGetFramework nuGetFramework)
         {
-            return nuGetFramework == null ? false : nuGetFramework.GetShortFolderName().StartsWith("portable-");
+            return nuGetFramework == null ? false : nuGetFramework.Framework == FrameworkConstants.FrameworkIdentifiers.Portable ||
```

Comment:
Wouldn't the starts with already cover this other case? If not, what is the FrameworkConstants.FrameworkIdentifiers.Portable value?

Reply:
no, NuGet started using portable45- and portable46- depending on how we construct the NuGetFramework type (if it's parsed from a string it doesn't get a version, when constructed from a FrameworkName it has the version).  Its potentially a bug that they're including version in portable tfms, but they've been doing it for a couple releases.